### ISSUE TITLE
PR Test Failure: A stable release of a package should not have on a prerelease dependency. Either modify the version spec of dependency "System.Console (ò 4.0.0-beta-23123)" or update the version field.

### ISSUE THREAD

**Issue by Pilchie** [dotnet/roslyn#6438]
*See http://dotnet-ci.cloudapp.net/job/dotnet_roslyn_prtest_win_dbg_eta/1507/ for more details.*

```
MSBuild auto-detection: using msbuild version '14.0' from 'C:\Program Files (x86)\MSBuild\14.0\bin'.
  Attempting to build package from 'Microsoft.CodeAnalysis.Scripting.Common.nuspec'.
  A stable release of a package should not have on a prerelease dependency. Either modify the version spec of dependency "System.Console (ò 4.0.0-beta-23123)" or update the version field.
d:\j\workspace\dotnet_roslyn_prtest_win_dbg_eta\Closed\Setup\NuGet\NuGet.proj(106,5): error MSB3073: The command "..\..\..\Open\NuGet Pack NuSpec\Microsoft.CodeAnalysis.Scripting.Common.nuspec -BasePath d:\j\workspace\dotnet_roslyn_prtest_win_dbg_eta\Open\Binaries\Debug\ -OutputDirectory d:\j\workspace\dotnet_roslyn_prtest_win_dbg_eta\Open\Binaries\Debug\NuGet\Release -prop licenseURL=http://go.microsoft.com/fwlink/?LinkId=529443 -prop binaries=d:\j\workspace\dotnet_roslyn_prtest_win_dbg_eta\Open\Binaries\Debug\ -prop currentVersion="1.1.0" -prop corextVersion="1.1." -prop microsoftCodeAnalysisAnalyzersVersion=1.0.0 -prop systemReflectionMetadataVersion=1.1.0 -prop systemCollectionsImmutableVersion=1.1.37 -prop authors=Microsoft -prop projectURL=http://msdn.com/roslyn -prop releaseNotes="Preview of Microsoft .NET Compiler Platform ("Roslyn")" -prop tags="Roslyn CodeAnalysis Compiler CSharp VB VisualBasic Parser Scanner Lexer Emit CodeGeneration Metadata IL Compilation Scripting Syntax Semantics" -prop buildName=" exited with code 1.
```

**Comment by Pilchie** [comment]
Tagging @tmat

**Comment by tannergooding** [comment]
@Pilchie, the packages have been moved to only create the PreRelease packages again.

@tmat, has a PR out here: https://github.com/dotnet/roslyn/pull/6437, to remove the dependency and we are working with CoreFX to see if we can get a release package made available.

**Comment by ManishJayaswal** [comment]
Adding label approved for next preview as the PR was already approved.

**Comment by tannergooding** [comment]
This should be resolved now (stabilization, working on merging to master).

**Comment by tmat** [comment]
Fixed by  #8bebee5
